### DISCUSSION TITLE
Can I change it manually to 1.20.4?

### DISCUSSION THREAD

**Discussion by PeterWritesCode** [ORelio/Minecraft-Console-Client#2661] [Q&A] [answered]
Hey guys. I want to use this for 1.20.4, but it tells me it doesn't work, and only for 1.20.2.

Is there any way I can change this? Like make it work for 1.20.4, like drop the 1.20.4 .jar somewhere or something?

Thanks!

**Comment by milutinke** [comment]
> Hey guys. I want to use this for 1.20.4, but it tells me it doesn't work, and only for 1.20.2.
> 
> Is there any way I can change this? Like make it work for 1.20.4, like drop the 1.20.4 .jar somewhere or something?
> 
> Thanks!

1.20.4 is not supported yet, there is a lot of work to be done for it to be supported as the way of how the server sends the chat and some other info has changed and we need to figure it out and update it.
For now you could use a Proxy that lets you connect from 1.20.2 to 1.20.2.

**Reply by PeterWritesCode** [reply]
Hello, thanks! I didn't know that those kind of changes were made from 1.20.2 to 1.20.4, I apologise. 
What do you mean by a proxy? Just the MCC for 1.20.2? 
Thank you

**Reply by milutinke** [reply, marked as answer]
I mean specifically a minecraft proxy that allows you to connect from one version to another, not the proxy support in the MCC.

**Reply by PeterWritesCode** [reply]
Thank you so much, I had no idea such a thing existed. That is truly awesome!
Now I can make a bot to load my farm chunks ahaha

**Reply by milutinke** [reply]
The 1.20.4 version is now officially supported.
